SU-HSDO/suhumsci
# HSD8-579 Use ultimate cron to break up migration importers

# READY FOR REVIEW

# Summary
- use ultimate cron to create separate cron tasks for each migration group.
- Cron queue was not a viable solution since we want these items to execute on every cron. cron queue doesn't do that.

# Needed By (Date)
- 2/13

# Urgency
- medium

# Steps to Test
1. Checkout this branch
1. `composer install --prefer-source`
1. sync a site like philit `blt drupal:sync --site=philit --partial`
1. view the page `/admin/config/system/cron/jobs` and ensure 3 cron jobs for "H&S Migrate" module category.
1. run each of the migration cron jobs and ensure they are successful (Capx will take a bit of time)
1. run cron from drush and ensure it's successful
1. review the `/admin/config/system/cron/jobs` page again to ensure no errors on any cron job.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
